### ISSUE TITLE
Nuscenes

### ISSUE THREAD

**Issue by JasonMa2016** [OATML/carsuite#12]
Hi authors,

Thank you very much for open-sourcing your code for the greater community. I am wondering if there are plans to also release codebase used for the Nuscenes portion of the paper? In particular, I am wondering if there are implementations of the models that would work with Nuscenes. As far as I know, the current repository only contains models interfacing CARLA.

**Comment by filangelos** [comment]
Hi @JasonMa2016,

The [`oatmobile.baselines`](https://github.com/OATML/oatomobile/tree/alpha/oatomobile/baselines/torch) should be easily adaptable to [`nuscenes.prediction`](https://github.com/nutonomy/nuscenes-devkit/tree/master/python-sdk/nuscenes/prediction/tests) API.

We didn't plan to release this since `oatomobile` is focused on CARLA (i.e., control tasks) and not that much on prediction tasks but we may take this into account! In the meanwhile, let me know if you have any trouble adapting the code and we can give you a hand. Also, take a look at [PRECOG's codebase](https://github.com/nrhine1/precog/tree/master/precog/ext/nuscenes) on how to preprocess NuScenes LIDAR point-clouds.

**Comment by JasonMa2016** [comment]
Thanks for the pointer! I wonder if OATMOBILE's (unreleased) code base for nuscenes utliizes PRECOG's codebase for preprocessing NuScenes LIDAR point-clouds?

**Comment by filangelos** [comment]
Yes! But only the LIDAR preprocessing methods, we didn't use any of the multi-agent modelling PRECOG does!

**Comment by JasonMa2016** [comment]
I see, thank you! I wonder if the architecture detail and training hyperparameters for the Nuscenes version of your methods can be shared?

**Comment by filangelos** [comment]
It is the same [one](https://github.com/OATML/oatomobile/blob/alpha/oatomobile/baselines/torch/dim/model.py#L36) used in this repo, by adapting the output shape and input modalities accordingly.

**Comment by JasonMa2016** [comment]
Thanks for the response! Sorry to bother again, but I noticed the OATOMOBILE utilizes features such as "is_at_traffic_light" and "traffic_light_state" in computing the visual features for the models. I wonder if those information are used in the NuScenes versions of the models, and if so, how are they extracted? I am having trouble finding relevant code in PRECOG's codebase. Thank you!

**Comment by filangelos** [comment]
No, we just use the LIDAR observation for the NuScenes experiments.

**Comment by JasonMa2016** [comment]
Thank you! These are all the questions I had.
